libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/extensions/Mappers.kt

```diff
@@ -0,0 +1,31 @@
+@file:JvmName("Mappers")
+
+package com.mapbox.services.android.navigation.v5.utils.extensions
+
+import com.mapbox.api.directions.v5.WalkingOptions
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteLeg
+import com.mapbox.navigation.base.route.model.Route
+import com.mapbox.navigation.base.route.model.WalkingOptionsNavigation
+
+fun WalkingOptionsNavigation.mapToWalkingOptions(): WalkingOptions = WalkingOptions
+    .builder()
+    .walkingSpeed(walkingSpeed)
+    .walkwayBias(walkwayBias)
+    .alleyBias(alleyBias)
+    .build()
+
+fun Route.mapToDirectionsRoute(): DirectionsRoute {
```

Comment:
Should we also include this mapper into `Mappers.kt` (`libdirections-offboard`) as well?

Reply:
I'm writing my own mappers, no need

Reply:
Not this one. We already exposed `DirectionsRoute.mapToRoute()` https://github.com/mapbox/mapbox-navigation-android/blob/23708f1180a534852f6c994fc24b370f8acd0bf3/libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/extension/Mappers.kt#L13 but not the other way around `Route.mapToDirectionsRoute()` https://github.com/mapbox/mapbox-navigation-android/blob/cd1e9141a9cc0ebb3183e79eff6515fb5990d79c/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/extensions/Mappers.kt#L18

cc @RingerJK @Lebedinsky

Reply:
We can leave this duplication as we will be deprecating these `Mappers`. We're good here 🚀